File: src/main/java/no/nav/pensjon/regler/domain/beregning2011/IGRegulering.java
package no.nav.pensjon.regler.domain.beregning2011;

public interface IGRegulering extends IJustering {

    public int getForrigeG();

    public void setForrigeG(int forrigeG);

    public int getGjeldendeG();

    public void setGjeldendeG(int gjeldendeG);
}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/IGRegulering.kt
package no.nav.pensjon.regler.domain.beregning2011

import com.fasterxml.jackson.annotation.JsonSubTypes
import com.fasterxml.jackson.annotation.JsonTypeInfo

@JsonSubTypes(
    JsonSubTypes.Type(value = GReguleringDetaljer::class)
)
@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
interface IGRegulering : IJustering {
    var forrigeG: Int
    var gjeldendeG: Int
}